File: tbank-cooldown-api/src/main/java/org/svids/tbankcooldownapi/entity/PurchaseCategory.java
package org.svids.tbankcooldownapi.entity;

public enum PurchaseCategory {
    ELECTRONICS("Электроника"),
    CLOTHING("Одежда"),
    FOOD("Еда"),
    HOME("Хозтовары"),
    OTHER("Другое");

    final String name;

    PurchaseCategory(String name) {
        this.name = name;
    }
}

File: tbank-cooldown-client/tbank-cooldown-client/Enums/PurchaseCategory.swift
//
//  PurchaseCategory.swift
//  tbank-cooldown-client
//
//  Created by Александр Павлицкий on 06.12.2025.
//

import Foundation

// MARK: - PurchaseCategory
enum PurchaseCategory: String, CaseIterable, Hashable {
    case all = "Все"
    case electronics = "Электронника"
    case clothing = "Одежда"
    case food = "Еда"
    case home = "Дом"
    case other = "Другое"
}
